### ISSUE TITLE
Example "terminator-split -m 8" does not work

### ISSUE THREAD

**Issue by MarcSeebold** [AlekseyChudov/terminator-split#5]
1) "-m" is not documented
2) Running "terminator-split -m 8" only opens one Terminator window for me (no splits at all)
Version:
terminator 1.91 on Ubuntu 18.04.1 LTS (4.15.0-36-generic), python --version: Python 2.7.15rc1

**Comment by acudovs** [comment]
"-m" option is terminator own option (documented as TERMINATOR_OPTIONS in help message)

```
        terminator-split [-h] [-d]
                         [-g CONFIG] [-t TERMINATOR] [-s SHELL]
                         [TERMINATOR_OPTIONS]
                         number
```

My terminator version has this option to maximise the window

```
$ terminator -v
terminator 0.98

$ terminator --help | grep -- -m
  -m, --maximise        Maximise the window
```

I have not tested the terminator version 1.X yet. I will check the new version when I have time.

**Comment by acudovs** [comment]
Fixed in 1258e047760ffed17c2934e8fc4b34040b33869b. Try the latest version.
